02_rand_dataset.ipynb

```diff
@@ -33,56 +24,400 @@
+       "    <tr>\n",
+       "      <th>kabul</th>\n",
+       "      <td>3.0</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>57.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>kirkuk</th>\n",
+       "      <td>5.0</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>mosul</th>\n",
+       "      <td>16.0</td>\n",
+       "      <td>4.0</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>pakistan</th>\n",
+       "      <td>6.0</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>14.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>palestine</th>\n",
+       "      <td>2.0</td>\n",
+       "      <td>4.0</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>3.0</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "             al - qaeda  bilad al - rafidayn  fatah  hamas   plo  taliban\n",
+       "afghanistan         6.0                  NaN    NaN    NaN   NaN    260.0\n",
+       "baghdad            20.0                 31.0    NaN    NaN   NaN      NaN\n",
+       "basra               NaN                  4.0    NaN    NaN   NaN      NaN\n",
+       "beirut              NaN                  NaN    1.0    1.0  11.0      NaN\n",
+       "gaza                NaN                  NaN   11.0   70.0   NaN      NaN\n",
+       "iraq               53.0                 23.0    1.0    NaN   6.0      NaN\n",
+       "israel              1.0                  NaN   16.0   21.0  19.0      NaN\n",
+       "istanbul            3.0                  NaN    NaN    NaN   NaN      NaN\n",
+       "kabul               3.0                  NaN    NaN    NaN   NaN     57.0\n",
+       "kirkuk              5.0                  NaN    NaN    NaN   NaN      NaN\n",
+       "mosul              16.0                  4.0    NaN    NaN   NaN      NaN\n",
+       "pakistan            6.0                  NaN    NaN    NaN   NaN     14.0\n",
+       "palestine           2.0                  4.0    NaN    1.0   3.0      NaN"
+      ]
+     },
+     "execution_count": 71,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Transform the dictionary into a pandas DataFrame and fill NaN values with zeroes\n",
+    "le_df = pd.DataFrame.from_dict(dict(location_entity_dict),dtype=int)\n",
```

Comment:
Could you please add space after comma `pd.DataFrame.from_dict(dict(location_entity_dict),dtype=int)` -->
`pd.DataFrame.from_dict(dict(location_entity_dict), dtype=int)`